CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # CHANGELOG
 
+## 5.x
+* [patch] Add SSL support for API-based resources
```

Comment:
this is more of a [minor], as you are adding new functionality but in a backward compatible way.